### PR TITLE
EZEE-2790: Impl. event actions

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
Added:
- /calendar/event GET
- /calendar/event/grouped-by-day GET
- /calendar/event/{eventType} POST

### CODE REVIEW

File: input/ez-types.raml [hunk 1]
```diff
@@ -592,9 +592,77 @@ Criterion:
...
+            type: DateTime
+        type:
+            description: 'Event type identifier or name.'
+            type: string
+        name:
+            description: 'Name of the calendar event.'
+            type: string
+#        attributes:
+#            description: 'List of event attributes.'
+#            type: EventAttribute[]
```
Comment (adamwojs) [inline input/ez-types.raml]: This should be uncommented.

File: input/ez-types.raml [hunk 2]
```diff
@@ -592,9 +592,77 @@ Criterion:
...
+            type: Event[]
+#EventAttribute:
+#    type: object
+#    properties:
+#        label:
+#            description: 'Label of an event.'
+#            type: string
+#        value:
+#            description: ''
+#            type: string
```
Comment (adamwojs) [inline input/ez-types.raml]: This should be uncommented.

**Review by adamwojs — APPROVED** [review]

File: input/ez-calendar.raml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        description: Calendar event list.
+        queryParameters:
+            start:
+                description: Query start date.
+                type: DateTime
+            end:
+                description: Query end date.
+                type: DateTime
+            types:
+                description: The types of events that will be displayed.
```
Comment (DominikaK) [inline input/ez-calendar.raml]:
```suggestion
                description: The types of events that are displayed.
```

File: input/ez-calendar.raml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                type: string
+            languages:
+                description: Language code. Restricts the output of translatable Fields to the given languages.
+                type: string
+            count:
+                description: Number of parameters that will be returned in the list.
```
Comment (DominikaK) [inline input/ez-calendar.raml]:
```suggestion
                description: Number of parameters that are returned in the list.
```

File: input/ez-calendar.raml [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+            description: Calendar event list grouped by day.
+            queryParameters:
+                start:
+                    description: Query start date.
+                    type: TimeStamp
+                end:
+                    description: Query end date.
+                    type: TimeStamp
+                types:
+                    description: The types of events that will be displayed.
```
Comment (DominikaK) [inline input/ez-calendar.raml]:
```suggestion
                    description: The types of events that are displayed.
```

File: input/ez-calendar.raml [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+                    type: string
+                languages:
+                    description: Language code. Restricts the output of event list to the given languages.
+                    type: string
+                count:
+                    description: Number of parameters that will be returned in the list.
```
Comment (DominikaK) [inline input/ez-calendar.raml]:
```suggestion
                    description: Number of parameters that are returned in the list.
```

**Review by DominikaK — APPROVED** [review]

**Pull request merged**